api/openapi-spec/swagger.json

```diff
@@ -680,13 +680,6 @@
        "name": "fieldSelector",
        "in": "query"
       },
-      {
-       "uniqueItems": true,
-       "type": "boolean",
-       "description": "If true, partially initialized resources are included in the response.",
-       "name": "includeUninitialized",
```

Comment:
These `includeUninitialized`'s are moved to Path-level parameters (one level higher) from Operation-level parameters, because all operations under the path use `includeUninitialized`